packages/inscription-view/src/App.tsx

```diff
@@ -21,11 +21,16 @@ import { useTranslation } from 'react-i18next';
 function App({ outline, app, pmv, pid }: InscriptionElementContext & InscriptionOutlineProps) {
   const { t } = useTranslation();
   const [context, setContext] = useState({ app, pmv, pid });
-  const [initData, setInitData] = useState<Record<string, ElementData>>({});
+  const [initData, setInitData] = useState<ElementData | undefined>(undefined);
```

Comment:
you don't have to put undefined as param if the type sais so
```suggestion
  const [initData, setInitData] = useState<ElementData | undefined>();
```